setup.cfg

```diff
@@ -17,3 +17,5 @@ use_parentheses = true
 lines_after_imports = 2
 default_section = THIRDPARTY
 sections = FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
+known_first_party = fv3core
+known_third_party = f90nml,pytest,xarray,numpy,mpi4py,gt4py
```

Comment:
@mcgibbon I'm not sure if we need all these. Can you check them?

Reply:
Some of those may only be used through sub-packages.

Reply:
Even if we never import them (we shouldn't import mpi4py directly so the package can run in serial without MPI, and you should probably remove that one), there's no real harm to adding extra packages to this list. From what I can tell, it's only used for isort.

I would consider `fv3gfs` (e.g. `fv3gfs.util`) a first-party package.

Reply:
`fv3gfs` still needs to be added to first party imports.